src/components/RegisterProgram/index.tsx

```diff
@@ -24,18 +28,21 @@ const RegisterProgram: FC<RegisterProgramProps> = ({
   onClose,
   programId
 }) => {
-  const { data: program } = useGetProgram({ programId })
-
-  const { branch, endDate, startDate, corporatePrice, programHeader } =
-    program || {}
-
   const [activeStep, setActiveStep] = useState<number>(0)
 
-  const formik = useFormikRegister({ onClose, activeStep, setActiveStep })
+  const { data: voucher, isLoading: loadingVoucher } = useGetVoucherQuery()
+  const { data: program, isLoading: loadingProgram } = useGetProgram({
+    programId
+  })
+  const {
+    mutateAsync: corporateTokenTransaction,
+    isLoading: loadingTokenTransaction
```

Comment:
karna ini mutate function, tambahin prefix `create` / `update` di namingnya

```suggestion
    mutateAsync: createCorporateTokenTransaction,
    isLoading: loadingCreateTokenTransaction
```